src/collections/components/CollectionSearchForm/CollectionSearchForm.validation.tsx

```diff
@@ -4,10 +4,12 @@ import { CollectionStatus } from '../../../api/generatedTypes';
 export const validationSchema = yup.object({
   title: yup.string().min(3),
   author: yup.string().min(3),
+  labels: yup.array(),
   status: yup.mixed<CollectionStatus>().oneOf(Object.values(CollectionStatus)),
-  filterRequired: yup.bool().when(['title', 'author', 'status'], {
-    is: (title: any, author: any, status: any) => !title && !author && !status,
-    then: yup.bool().required('at least one filter is required'),
+  filterRequired: yup.bool().when(['title', 'author', 'status', 'labels'], {
+    is: (title: any, author: any, status: any, labels: any) =>
+      !title && !author && !status && labels.length < 1,
+    then: yup.bool().required('At least one filter is required.'),
     otherwise: yup.bool(),
```

Comment:
The custom validation error has been updated to include the Labels field - if there's at least one label there, the search will be executed in the absence of other filters. All the old functionality is preserved - e.g. can still search just for authors, etc.

Reply:
nice!